tango_simlib/tests/DishElementMaster_SIMDD.json

```diff
@@ -190,6 +190,70 @@
           "quantity_simulation_type": "ConstantQuantity"
         }
       }
+    },
+    {
+      "basicAttributeData": {
```

Comment:
- [ ] Is most of this information not in the xmi file? Why repeat it in the simdd?